etl-pipeline/processes/grin/unpack.py

```diff
@@ -0,0 +1,46 @@
+import os
+import tempfile
+import tarfile
+
+from processes.grin.download import GRINDownloadService
+from logger import create_log
+
+logger = create_log(__name__)
+
+
+class GRINUnpackService:
```

Comment:
Hypothetically I think we could rework this to avoid writing out to the temporary directory entirely here, but that is a pretty major change and could also open us up to a memory leak (which is currently handled by the `with` block) so I think let's leave this as is for now, but could you add a #TODO here or something to look at that? I think it could be faster in the future to avoid the disk read/writes